x-pack/plugins/security_solution/server/lib/entity_analytics/risk_engine/migrations/update_risk_score_mappings.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { asyncForEach } from '@kbn/std';
+import { first } from 'lodash/fp';
+import type { EntityAnalyticsMigrationsParams } from '../../migrations';
+import { RiskEngineDataClient } from '../risk_engine_data_client';
+import { getDefaultRiskEngineConfiguration } from '../utils/saved_object_configuration';
+import { RiskScoreDataClient } from '../../risk_score/risk_score_data_client';
+import type { RiskEngineConfiguration } from '../../types';
+import { riskEngineConfigurationTypeName } from '../saved_object';
+import { buildScopedInternalSavedObjectsClientUnsafe } from '../../risk_score/tasks/helpers';
+
+export const MAX_PER_PAGE = 10_000;
```

Comment:
Should we handle the scenario where users have more than 10.000 spaces where the risk engine is installed? 🤔

Reply:
nah https://discuss.elastic.co/t/max-number-of-spaces-and-roles-in-kibana/316933